recipes-iec61850/libiec61850/libiec61850.bb

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2021, Alliander (http://www.alliander.com)
+# SPDX-License-Identifier: Apache-2.0
+
+LICENSE = "GPL-3.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
+
+SRC_URI = "git://github.com/mz-automation/libiec61850.git;branch=v1.4"
+
+# PV = "v1.4+git${SRCPV}" # TODO check if this should be included
+SRCREV = "${AUTOREV}"
```

Comment:
AUTOREV means that you could have a different version between 2 builds depending if something new was pushed to libiec61850.
We tend to avoid that specially when dealing with external source code because your software could be altered without you specially expect it.

Reply:
I was fighting a bit with the build system. Will have another look how to do it.

Reply:
this should be fixed now